### PR TITLE
Fix include issue in example null transform plugin and test

### PR DESCRIPTION
**Pull request by dyrock**
Fix null_transform.test.py on how to load plugin
Minor changes on gold file

Removed all `ATS_UNUSED`. This fix deals with the `#include "ts/ink_defs.h"`, which is internal and requires additional include argument to compile with tsxs. This causes problem in autest.

### CODE REVIEW

**Comment by persiaAziz** [comment]
[approve ci]

**Comment by shinrich** [comment]
[approve ci]

File: tests/tools/plugins/null_transform.c [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /** @file
```
Comment (SolidWallOfCode) [inline tests/tools/plugins/null_transform.c]: Why is this duplicated?

File: example/null_transform/null_transform.c [hunk 2]
```diff
@@ -183,7 +183,7 @@ handle_transform(TSCont contp)
 }
 
 static int
-null_transform(TSCont contp, TSEvent event, void *edata ATS_UNUSED)
+null_transform(TSCont contp, TSEvent event, void *edata)
```
Comment (SolidWallOfCode) [inline example/null_transform/null_transform.c]: Are these actually unused? If so, drop the name.

File: tests/tools/plugins/null_transform.c (hunk 1 above)
Comment (dyrock) [reply tests/tools/plugins/null_transform.c]: The plugin file is copied over to tests/tools/plugins directory to be built from within the test.

**Pull request merged**